source/nxp/lpc54018/FlashDev.c

```diff
@@ -0,0 +1,34 @@
+/* CMSIS-DAP Interface Firmware
+ * Copyright (c) 2009-2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "FlashOS.H"        // FlashOS Structures
+
+#define FLASH_DRV_VERS (0x0100+VERS)   // Driver Version, do not modify!
+
+struct FlashDevice const FlashDevice = {
+    FLASH_DRV_VERS,               // Driver Version, do not modify!
+    "LPC54608 16MB Serial Flash", // Device Name
```

Comment:
Should the device name be changed to "LPC54018"? Probably doesn't matter except for technical correctness.

Reply:
Thanks, I have made the fix